package.json

```diff
@@ -28,6 +28,9 @@
     "chai-subset": "^1.6.0",
     "semver": "^7"
   },
+  "peerDependencies": {
```

Comment:
Why not a devDependency?  peerDependencies is for plugins, but cap-samples is an app.
Also, why not open it up to eslint 9, e.g. use `>=8` ?